.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "tunnistamo"]
 	path = tunnistamo
-	url = https://github.com/City-of-Helsinki/tunnistamo
+	url = ./tunnistamo/
```

Comment:
I don't think this works. You should add the submodule back with `git submodule add https://github.com/City-of-Helsinki/tunnistamo`